src/styles/layout/blog/article.scss

```diff
@@ -135,3 +135,41 @@
     gap: 12px;
   }
 }
+
+@include mixins.laptop {
+  .article {
+    max-width: 792px;
+    &__footer {
+      flex-wrap: wrap;
+    }
+  }
+}
+@include mixins.tablet {
+  .article {
+    max-width: 550px;
+    width: 100%;
+
+    &__images {
+      flex-direction: column;
+      display: block;
+      object-fit: cover;
+    }
+  }
+}
+
+@include mixins.phone {
+  .article {
+    max-width: 328px;
+    width: 100%;
+    &__images {
+      flex-direction: column;
+      display: block;
+      object-fit: cover;
```

Comment:
Could you check please, I think these styles will be working from previous our media